tox.ini

```diff
@@ -1,30 +1,27 @@
 [tox]
-envlist = py27,py35,py37
+envlist = {py27,py35,py37}{,-gui}
```

Comment:
This looks super obscure, would prefer an explicit list of everything instead so non-tox experts and quickly grok this.